.env

```
@@ -35,6 +35,7 @@ VITE_FEATURE_ARBITRUM_BRIDGE=true
 VITE_FEATURE_CUSTOM_TOKEN_IMPORT=true
 VITE_FEATURE_USDT_APPROVAL_RESET=true
 VITE_FEATURE_PORTALS_SWAPPER=true
+VITE_FEATURE_NEAR_INTENTS_SWAP=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FEATURE_NEAR_INTENTS_SWAP key should go before the VITE_FEATURE_OPTIMISM key

(UnorderedKey)